app/controllers/chatrooms_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
 
   def my_messages
     # TODO
-    @chatrooms = Chatroom.all
+    @chatrooms = Chatroom.where("name LIKE ?", "%_#{current_user.id}_%").includes(:messages).order('messages.created_at DESC')
```

Comment:
I'm good to approve but I'd like to understand what the ? in name LIKE does. Is it a wildcard? or does it specifically grab the current_user that you reference later?

Reply:
Sorry, I just remembered that this is to avoid SQL injections, forgot about that.

Reply:
Oh yeah! Exactly. To be honest I initially had it without the ?, and then checked the docs and decided to add it here 😆

Reply:
But it's that: a reference to the current_user.id